src/demo/index.html

```diff
@@ -155,6 +165,8 @@ <h1>Symphony Electron API Demo</h1>
                 activate: 'activate',
                 registerBoundsChange: 'register-bounds-change',
                 registerProtocolHandler: 'register-protocol-handler',
+                registerLogRetriever: 'register-log-retriever',
+                logReceiver: 'log-receiver',
```

Comment:
Shouldn't the operation be a verb? I think `send-logs` would be better.

Reply:
Yes, I'll go with send-logs